website/versioned_docs/version-0.10.0/accordproject-specification.md

```diff
@@ -97,7 +97,7 @@ The first step in converting the late delivery and penalty clause to use Accord
 
 These data elements comprise the Template Model for the clause. The template model is critical in that it defines formal semantics for the clause, and it is a locale neutral representation of the data that a template requires. It also enables powerful search, filtering and organization of templates, for example by finding all templates related to concept X, or all templates that can process a request of type Y.
 
-They are captured formally using the [Concerto modeling language](https://github.com/hyperledger/composer-concerto) (CML). CML is a lightweight schema language that defines namespaces, types, and relationships between types. It includes first-class support for modeling participants (individuals or companies), assets, transactions, enumerations, concepts, and events, and includes the typical features of an Object Oriented modeling language, including inheritance, meta-annotations (decorators), and field specific validators. CML also defines a serialization of instances to JSON and a validator for instances, making it easy to integrate with a wide variety of JSON-capable external systems.
+They are captured formally using the [Concerto modeling language](https://github.com/accordproject/concerto) (CML). CML is a lightweight schema language that defines namespaces, types, and relationships between types. It includes first-class support for modeling participants (individuals or companies), assets, transactions, enumerations, concepts, and events, and includes the typical features of an Object Oriented modeling language, including inheritance, meta-annotations (decorators), and field specific validators. CML also defines a serialization of instances to JSON and a validator for instances, making it easy to integrate with a wide variety of JSON-capable external systems.
```

Comment:
This is a static versioned file and shouldn't be edited, please remove this change.

When we make a new release of the documentation, Docusarus creates these `versions_docs` automatically as a snapshot of the documentation for historical reference.